src/cheri/riscv-priv-integration.adoc

```diff
@@ -435,76 +435,78 @@ endif::[]
+When using a system with {cheri_default_ext_name}, it may be desirable to disable CHERI register and instruction access to some (or all) privilege levels such that they operate as a RV32I/RV64I system without any observable presence of CHERI features.
 {cheri_priv_m_reg_enable_ext} includes functions to disable explicit access to CHERI
-registers and instructions.
-The {cheri_priv_m_reg_enable_ext} extension makes the `CRE` bit of <<mseccfg>>, <<menvcfg>>, and <<senvcfg>> writable.
+registers and instructions (hereafter referred to _disabling CHERI_).
+
+NOTE: When _CHERI is disabled_ for a specific privilege level, <<pcc>> and <<ddc>> bounds are _still_ enforced.
+
+The {cheri_priv_m_reg_enable_ext} extension makes the {CRE} bit of <<misa>>, <<menvcfg>>, and <<senvcfg>> writable to allow CHERI to be disabled.
 
 IMPORTANT: If {cheri_base_ext_name} is supported and {cheri_default_ext_name} is not supported, then {cheri_priv_m_reg_enable_ext} must also not be supported.
-In this case all CRE bits are hardwired to 1 and access to CHERI registers is always permitted.
+In this case all {CRE} bits are hardwired to 1 and CHERI is always enabled.
 This allows implementing a hart that always runs in {cheri_cap_mode_name}.
 
-CHERI register access is disabled if
+CHERI is disabled if
 
 * XLEN in the current mode is less than MXLEN, or
 * the endianness in the current mode is not the reset value of <<mstatus_cheri>>.MBE, or
-* the effective CRE for the current privilege is 0.
+* the effective CHERI-enable for the current privilege is 0.
 
-The effective CRE for the current privilege is:
+The effective CHERI-enable for the current privilege is:
 
-* Machine: `<<mseccfg>>.CRE`
-* Supervisor: `<<mseccfg>>.CRE & <<menvcfg>>.CRE`
-* User: `<<mseccfg>>.CRE & <<menvcfg>>.CRE & <<senvcfg>>.CRE`
+* Machine: `<<misa>>.{CRE}`
+* Supervisor: `<<misa>>.{CRE} & <<menvcfg>>.{CRE}`
+* User: `<<misa>>.{CRE} & <<menvcfg>>.{CRE} & <<senvcfg>>.{CRE}`
 
-NOTE: The effective CRE is always 1 in debug mode.
+NOTE: The effective CHERI-enable is always 1 in debug mode.
 
-NOTE: On reset CHERI register access is disabled (<<mseccfg>>.CRE resets to zero).
+NOTE: On reset CHERI is always disabled for backwards compatibility (<<misa>>.{CRE} resets to zero, <<ddc>> and <<pcc>> bounds are infinite).
 
-The following occurs when executing code in a privilege mode that has CHERI register access disabled:
+The following occurs when executing code in a privilege mode that has CHERI disabled:
 
-* Instructions that access full capability registers instead of only the XLEN subset (implicitly or explicitly) cause illegal instruction exceptions
+* Instructions that access full capability registers instead of only the XLEN subset (implicitly or explicitly) cause illegal instruction exceptions.
 +
-NOTE: The only instruction added by {cheri_base_ext_name} that does not access capability state is <<CRAM>>, all others are disabled.
```

Comment:
I'e removed this on purpose - it conflicts with text below and means we'll have to declare an explicit extension for this one instruction which is really not worth it